sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -229,6 +229,8 @@ class AdaptiveQueryExecSuite
       assert(smj.size == 3)
       val bhj = findTopLevelBroadcastHashJoin(adaptivePlan)
       assert(bhj.size == 2)
+      checkNumLocalShuffleReaders(adaptivePlan, 1)
```

Comment:
there are 2 broadcast joins, but we only have 1 local shuffle reader. Why is it?

Reply:
`checkNumLocalShuffleReaders` method should call `collect(plan)` not call `plan.collect`. I have updated and check all the local shuffle reader number in existing test methods. Thanks.